src/current/_includes/releases/v24.1/v24.1.0.md

```diff
@@ -237,11 +237,21 @@ In CockroachDB Self-Hosted, all available features are free to use unless their
 
 </div>
 
+<h4 id="v24-1-0-enterprise-features" markdown="1">Enterprise features</h4>
+
+{{site.data.alerts.callout_info}}
+The following are [enterprise-only](https://www.cockroachlabs.com/docs/v24.1/enterprise-licensing) features. [Request a 30-day trial license](https://www.cockroachlabs.com/get-cockroachdb/enterprise/) to try them out.
+{{site.data.alerts.end}}
+
+- The [`READ COMMITTED`]({% link v24.1/read-committed.md %}) isolation level now requires the cluster to have a valid [enterprise license]({% link v24.1//licensing-faqs.md %}#obtain-a-license). Otherwise, transactions which are configured to run as `READ COMMITTED` will be upgraded to [`SERIALIZABLE`]({% link v24.1/demo-serializable.md %}), as described in [Backward-incompatible changes](#read-committed-transaction-upgrade). [#120154](https://github.com/cockroachdb/cockroach/pull/120154)
```

Comment:
I don't think the backward-incompatible section should be linked here. What that other note is saying is that for enterprise users on 24.1, `READ COMMITTED` will _not_ be auto-upgraded to `SERIALIZABLE` because the cluster setting to enable Read Committed is on by default. I suggested a clarification to that note as well.

```suggestion
- The [`READ COMMITTED`]({% link v24.1/read-committed.md %}) isolation level now requires the cluster to have a valid [enterprise license]({% link v24.1//licensing-faqs.md %}#obtain-a-license). Without an enterprise license, [all transactions run as `SERIALIZABLE`]({% link v24.1/transactions.md %}#aliases). [#120154](https://github.com/cockroachdb/cockroach/pull/120154)
```